sql/parser2/eval.go

```diff
@@ -710,3 +750,43 @@ func evalTupleEQ(ldatum, rdatum Datum) (Datum, error) {
 	}
 	return dbool(true), nil
 }
+
+func evalTupleIN(arg, values Datum) (Datum, error) {
+	if arg == null {
+		return dbool(false), nil
+	}
+
+	vtuple := values.(dtuple)
+
+	if _, ok := arg.(dtuple); !ok && false {
```

Comment:
I thought that's what the comment on the next line was about. Guess I need to make it clearer.